symphony/content/content.blueprintsdatasources.php

```diff
@@ -1168,6 +1168,8 @@ public function __formAction()
             $this->_errors['name'] = __('This is a required field');
         } elseif (strpos($fields['name'], '\\') !== false) {
             $this->_errors['name'] = __('This field contains invalid characters') . ' (\\)';
+        } elseif (!preg_match('/^[a-z]/i', $fields['name'])) {
+            $this->_errors['name'] = __('The name of the data-source must begin with a letter.');
```

Comment:
Most of the time it's written "data source" (unhyphenated) in Symphony.

Reply:
Good catch!